docs/usage.md

```diff
@@ -3891,10 +3967,621 @@ nextflow clean -f -k
+--bamutils_clip_none_udg_left 3 \
+--bamutils_clip_none_udg_right 3 \
+--run_bedtools_coverage \
+--anno_file '../Reference/genome/Yersinia_pestis_C092_GCF_000009065.1_ASM906v1.gff' \
+--run_genotyping \
+--genotyping_tool 'ug' \
+--genotyping_source 'trimmed' \
+--gatk_ug_jar '../bin/GenomeAnalysisTK.jar' \
+--gatk_ploidy 2 \
+--gatk_ug_mode 'EMIT_ALL_SITES' \
+--gatk_ug_genotype_model 'SNP' \
+--run_multivcfanalyzer \
+--write_allele_frequencies \
+--min_base_coverage 5 \
+--min_allele_freq_hom 0.9 \
+--min_allele_freq_het 0.1 \
+--additional_vcf_files '../vcfs/*.vcf.gz'
+```
+
+For the two `min_allele_freq` parameters we specify that anything above 90%
+frequency is considered 'homozygous', and anything above 10% (but below 90%)
+is considered an ambiguous call and the frequency will be reported.
+
+With this, we are ready to submit! If running on a remote cluster/server, Make
+sure to run this in `screen` session or similar, so that if you get a `ssh`
+signal drop or want to log off, Nextflow will not crash.
+
+#### Tutorial Pathogen Genomics - Results
+
+Assuming the run completed without any crashes (if problems do occur, check all
+against [#usage](#pipeline-options) all parameters are as expected, or check the
+[FAQ](#troubleshooting-and-faqs), we can now check our results in `results/`.
+
+##### Tutorial Pathogen Genomics - MultiQC Report
+
+In here there are many different directories containing different output files.
+The first directory to check is the `MultiQC/` directory. In here you should
+find a `multiqc_report.html` file. You will need to view this in a web browser,
+so I recommend either mounting your server to your file browser, or downloading
+it to your own local machine (PC/Laptop etc.).
+
+Once you've opened this you can go through each section and evaluate all the
+results. For example, I normally look for things like:
+
+General Stats Table:
+
+- Do I see the expected number of raw sequencing reads (summed across each set
+  of FASTQ files per library) that was requested for sequencing?
+- Does the percentage of trimmed reads look normal for aDNA, and do lengths
+  after trimming look short as expected of aDNA?
+- Does the Endogenous DNA (%) columns look reasonable (high enough to indicate
+  you have recieved enough coverage for downstream, and/or do you lose an
+  unusually high reads after filtering )
+- Does ClusterFactor or '% Dups' look high (e.g. >2 or >10% respectively -  high
+  values suggesting over-amplified or badly preserved samples (i.e. low
+  complexity).
+- Do you see an increased frequency of C>Ts on the 5' end of molecules in
+  the mapped reads?
+- Do median read lengths look relatively low (normally <= 100 bp) indicating
+  typically fragmented aDNA?
+- Does the Median coverage and percent >3x (or whatever you set) show sufficient
+  coverage for reliable SNP calls and that a good proportion of the genome is
+  covered indicating you have the right reference genome?
```

Comment:
Also that the decrease between the different % of the genome covered is gradual and not drastic